packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -1,41 +1,51 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import styles from './Card.module.css';
 import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import classed from '../../lib/classed';
 
 export enum RaisedLabelType {
   Hot = 'Hot',
   Pinned = 'Pinned',
+  Beta = 'Beta',
 }
 
 const typeToClassName: Record<RaisedLabelType, string> = {
   [RaisedLabelType.Hot]: 'bg-theme-status-error',
   [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-cabbage-40',
 };
 
 export interface RaisedLabelProps {
   listMode?: boolean;
   type: RaisedLabelType;
   description?: string;
+  className?: string | undefined;
 }
 
+export const RaisedLabelContainer = classed(
+  'div',
+  'temp relative group group-hover:transform-none group-hover:z-10 group-hover:transition-transform group-hover:ease-linear group-hover:transition-z-index group-hover:duration-100 group-hover:step-end',
```

Comment:
Is it possible to remove the `temp` now?
```suggestion
  'relative group group-hover:transform-none group-hover:z-10 group-hover:transition-transform group-hover:ease-linear group-hover:transition-z-index group-hover:duration-100 group-hover:step-end',
```

Reply:
Great spot, fixing now